tests/testthat/test_paramtest_aorsf_surv_aorsf.R

```diff
@@ -36,8 +40,11 @@ test_that("paramtest surv.aorsf predict", {
     "object", # handled internally
     "new_data", # handled internally
     "pred_horizon", # all times required for distr
+    "pred_aggregate", # handled internally
```

Comment:
Same as for the train parameters

Reply:
Same as above, will add a commit with this included.